go.mod

```diff
@@ -12,7 +12,6 @@ require (
 	github.com/ipfs/go-datastore v0.4.5
 	github.com/ipfs/go-ipfs-blockstore v0.1.0
 	github.com/ipfs/go-ipfs-blocksutil v0.0.1
-	github.com/ipfs/go-ipfs-ds-help v0.0.1
```

Comment:
I guess CI doesn't check for these yet, once we switch to the standardized CI in #36 hopefully these won't slip through